homeassistant/components/nina/config_flow.py

```diff
@@ -209,10 +209,10 @@ async def async_step_init(
                     await nina.getAllRegionalCodes()
                 )
             except ApiError:
-                errors["base"] = "cannot_connect"
+                return self.async_abort(reason="no_fetch")
             except Exception as err:  # noqa: BLE001
                 _LOGGER.exception("Unexpected exception: %s", err)
```

Comment:
```suggestion
                _LOGGER.exception("Unexpected exception")
```
exception already logs the error. Depending on where your logger is from you don't need the `noqa` here as per https://docs.astral.sh/ruff/rules/blind-except/